src/Components/Aspire.Azure.AI.OpenAI/ConfigurationSchema.json

```diff
@@ -31,88 +31,13 @@
                     "ClientOptions": {
                       "type": "object",
                       "properties": {
-                        "Diagnostics": {
-                          "type": "object",
-                          "properties": {
-                            "ApplicationId": {
```

Comment:
Logged https://github.com/Azure/azure-sdk-for-net/issues/45066 which is making these properties not configurable anymore.